src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -185,6 +185,9 @@ The .NET Foundation licenses this file to you under the MIT license.
       <_IlcConditionallyRootedAssemblies Include="@(ManagedAssemblyToLink)" Condition="%(ManagedAssemblyToLink.TrimMode) == 'copyused'" />
       <_IlcTrimmedAssemblies Include="@(ManagedAssemblyToLink)" Condition="%(ManagedAssemblyToLink.TrimMode) == 'link'" />
       <_IlcNoSingleWarnAssemblies Include="@(ManagedAssemblyToLink)" Condition="%(ManagedAssemblyToLink.TrimmerSingleWarn) == 'false'" />
+
+      <!-- always export unmanaged entry points from System.Private.CoreLib -->
+      <UnmanagedEntryPointsAssembly Include="System.Private.CoreLib" />
```

Comment:
This should be also added to https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/aot/ILCompiler/repro/repro.csproj since nothing is going to work without it.

Reply:
Thanks, I missed that. Added.